ecf/include/emc_cdate.h

```diff
@@ -0,0 +1,11 @@
+datedir=${DATAROOT}/emc_ecflow_pdy.date.$$.$RANDOM
```

Comment:
What is the use of this file? And why does any other job not need it?

Reply:
It is removed. Please do a page reload to see the update.